policyengine_us/reforms/local/nyc/stc/phase_out/nyc_school_tax_credit_with_phase_out.py

```diff
@@ -0,0 +1,105 @@
+from policyengine_us.model_api import *
+from policyengine_core.periods import period as period_
+
+
+def create_nyc_school_tax_credit_with_phase_out() -> Reform:
+    class nyc_school_tax_credit_phase_out(Variable):
+        value_type = float
+        unit = USD
+        entity = TaxUnit
+        label = "NYC School Tax Credit Phase Out"
+        definition_period = YEAR
+        defined_for = "in_nyc"
+        reference = "https://www.nysenate.gov/legislation/bills/2025/S2238"
+
+        def formula(tax_unit, period, parameters):
+            """
+            Calculates the phase-out amount for the NYC School Tax Credit.
+
+            Args:
```

Comment:
I meant a file-level docstring explaining the policy. These args do not match the formula args.

Reply:
Added a readme instead